CHANGELOG.md

```diff
@@ -2,9 +2,16 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
-## [3.3.2] -
+## [3.4.0] -
+### Added
+ - [Updated Prune-Depth Documentation. Add Unit Tests for more limit values](https://github.com/joyent/java-manta/pull/484)
+### Changed
+ - [Upgraded dependencies to latest stable versions:](https://github.com/joyent/java-manta/commit/1ea4694b2747c42fa7b8b60733fbd7699a3f2426)
 ### Fixed
- - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486) 
+ - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486)
+ - [Test Groups that include Heavyweight Tests that are not run by default](https://github.com/joyent/java-manta/issues/483) 
```

Comment:
Is this change included? I think it isn't, right?

Reply:
Removed.